core/src/test/java/bisq/core/user/PreferencesTest.java

```diff
@@ -124,7 +124,7 @@ public void testGetUniqueListOfCryptoCurrencies() {
 
         preferences.readPersisted();
 
-        assertEquals(13, preferences.getCryptoCurrenciesAsObservable().size());
+        assertEquals(14, preferences.getCryptoCurrenciesAsObservable().size());
```

Comment:
Do not touch this file

Reply:
If I don't change this, build/test will fail because the value returned from preferences.getCryptoCurrenciesAsObservable().size() is 14